torchbenchmark/models/BERT_pytorch/__init__.py

```diff
@@ -10,18 +10,63 @@
 from .bert_pytorch.model import BERT
 from torch.utils.data import DataLoader
 
-torch.manual_seed(1337)
-random.seed(1337)
-np.random.seed(1337)
-torch.backends.cudnn.deterministic = True
+torch.backends.cudnn.deterministic = False
 torch.backends.cudnn.benchmark = False
 from pathlib import Path
 from ...util.model import BenchmarkModel
 from torchbenchmark.tasks import NLP
+
+import io
+
+class CorpusGenerator(io.TextIOBase):
+
+    def __init__(self, words, lines):
+        self.lines_read = 0
+        self.lines = lines
+        self.words = words
+
+    def reset(self):
+        self.lines_read = 0
+
+    def readable(self):
+        if (self.lines_read > self.lines):
+            return False
+        return True
+
+    def readline(self):
```

Comment:
Maybe remove empty lines in this function to make it more compact? Also, it will be helpful to add comment on what each line looks like.